### PR TITLE
Hurumap Chart description & Sources

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
- [x] Add chart description for each geography with chart table information
- [x] Add sources

Fixes [#169618702](https://www.pivotaltracker.com/story/show/169618702)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Desktop Screenshots
Peek 2019-11-19 22-55

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: wp-content/plugins/hurumap/api.php [hunk 1]
```diff
@@ -10,6 +10,14 @@ function register_routes()
             'callback'              => 'get_charts'
         ),
     ));
+    //single hurumap
+    $endpoint_hurumap_chart = '/hurumap/(?P<chart_id>[\w\-]+)$';
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/api.php]: Shouldn't we reuse `/charts`?

File: wp-content/plugins/hurumap/src/HURUmapBlock/EditChart.js [hunk 2]
```diff
@@ -56,6 +56,12 @@ function EditChart({
         stat: {
           ...JSON.parse(chart.stat),
           queryAlias: `viz${index}`
+        },
+        description: {
+          ...JSON.parse(chart.description)
+        },
+        source: {
+          ...JSON.parse(chart.source)
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/HURUmapBlock/EditChart.js]: Wouldn't `source: JSON.parse(chart.source)` yield the same results?

**Review by kilemensi — COMMENTED** [review]
:+1: @KhadijaMahanga 

Lets add the geo filtering and description to preview window.

File: wp-content/plugins/hurumap/api.php (hunk 1 above)
Comment (KhadijaMahanga) [reply wp-content/plugins/hurumap/api.php]: yes, this is not used at all. I will remove it

**Comment by KhadijaMahanga** [comment]
@kilemensi this is also ready for another review

File: wp-content/plugins/hurumap/src/data/queries.js [hunk 3]
```diff
@@ -14,6 +14,17 @@ export const buildDataCountQuery = charts => gql`
   }
 `;
 
+export const tableGeoCountQuery = table => gql`
```
Comment (kilemensi) [inline wp-content/plugins/hurumap/src/data/queries.js]: We need to improve this names...

**Review by kilemensi — APPROVED** [review]
:rocket: @KhadijaMahanga 

------------------------

1. Sources are part of data entry. Just like we don't edit the entered data, the source (at least link) should not be editable there.
2. Lets just remove this title/heading...
Screenshot from 2019-11-21 13-25-00

**Pull request merged**